packages/broker/test/integration/broker-subscriptions.test.ts

```diff
@@ -26,6 +26,11 @@ const grantPermissions = async (streams: Stream[], brokerUsers: Wallet[]) => {
     }
 }
 
+export const getStreamParts = async (broker: Broker): Promise<StreamPartID[]> => {
+    const node = await (broker.getStreamrClient().getNode())
+    return node.getStreamParts()
+}
```

Comment:
Not necessarily within the scope of this PR (up to you): but could we get this info thru `client.getSubscriptions()` as well?

Reply:
Good point 👍